services/QuillLMS/client/app/bundles/Teacher/containers/StudentAccount.jsx

```diff
@@ -9,6 +9,20 @@ import getAuthToken from '../components/modules/get_auth_token';
 const GENERAL = 'general'
 const PASSWORD = 'password'
 
+function providerAccountType(googleId, cleverId, classlinkId) {
```

Comment:
Let's introduce a configuration hash to store this data rather than the position of arguments in a function (which is brittle and not generic).   See providerHelpers as an example of how to do this:
client/app/bundles/Teacher/components/classrooms/providerHelpers.ts

Also, the user the model has a method `def provider` which will give you the type of an account.  Then you only need to pass `provider` and `user_external_id` (kind of like a polymorphic object)